extensions-contrib/statsd-emitter/src/test/java/org/apache/druid/emitter/statsd/StatsDEmitterTest.java

```diff
@@ -181,7 +181,7 @@ public void testServiceAsTagOption()
             client
     );
     client.time("druid.query.time", 10,
-            "service:druid/broker", "dataSource:data-source", "type:groupBy", "hostname:brokerHost1"
+            "druidService:druid/broker", "dataSource:data-source", "type:groupBy", "hostname:brokerHost1"
```

Comment:
Is there any way to test that reserved tag keys are not used?

Reply:
The list of reserved is on this documentation: https://docs.datadoghq.com/tagging/

Testing that would mean adding the list of reserve words in this project, we will need to maintain that list to match the "source of truth" that is elsewhere (the doc above). So I'm not sure it worth it. What do you think ?

Reply:
I was wondering if there was an easy way to automatically get the list of reserved tag keys, but if not, then I don't think it's worth manually syncing something to the source of truth.

Reply:
AFAIK, there is no such list of tags we can import easily in this project :(